RPIPs/RPIP-1.md

```diff
@@ -66,6 +66,8 @@ The following is the standardization process for all RPIPs in all tracks:
 
 **Living** - A special status for RPIP that are designed to be continually updated and not reach a state of finality. This includes most notably RPIP-1.
 
+**Obsolete** - An Obsolete RPIP has been replaced, superceded or removed as a result of the passage of another RPIP.
```

Comment:
Hmmm... is this meant to only allow for obsolescence due to another RPIP? Wondering both whether it should be left flexible like "...removed, eg, as a result..." or if not whether we should make the language very explicitly ONLY for that case.

Reply:
Uhm. If its been ratified, you definitely don't want to obsolete it without some sort of vote, and currently that's just RPIPs. If its not ratified and its living informational... yeah I can see the argument for this not making sense.

Reply:
I cut down the definition to avoid the issue. Feels like to make it airtight it would be more more specific than the entire rest of RPIP-1, and I would rather tackle it all together at some point in the future.